lib/features/outline/index.js

```diff
@@ -1,10 +1,11 @@
 import Outline from './Outline';
-
+import MultiSelectionOutline from './MultiSelectionOutline';
 
 /**
  * @type { import('didi').ModuleDeclaration }
  */
 export default {
-  __init__: [ 'outline' ],
-  outline: [ 'type', Outline ]
+  __init__: [ 'outline', 'multiSelectionOutline' ],
```

Comment:
We now have to `__depend__` on the selection module. Otherwise the outline module will blow up if the selection module isn't added explicitly.

Reply:
Done via https://github.com/bpmn-io/diagram-js/pull/943/commits/ef8528e1116f5100329c375df8222eaf775f739a, including updating the tests.